builq.go

```diff
@@ -5,57 +5,50 @@ import (
 	"strings"
 )
 
-// Builder for SQL queries.
+// TODO(junk1tm): support PostgreSQL-style numbered placeholders.
+var Placeholder = "$"
```

Comment:
To fix that I'm proposing next API:
```
q := builq.New() // Postgres-like placeholders ($1, $2, etc)
q := builq.NewPlaceholder("?") // MySQL-like or any other DB/placeholder
```

The word `Builder` can be skipped 'cause package name is already about ~builder. WDYT ?

Reply:
Just pushed something like this, please take a look